app/src/components/integrations/integrationProviders/ldapIntegration/ldapFormFields/inputConditional/inputConditional.jsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Dropdown, FieldText } from '@reportportal/ui-kit';
+import {
+  CONDITION_CNT,
+  CONDITION_EQ,
+  CONDITION_NOT_CNT,
+  CONDITION_NOT_EQ,
+} from 'components/filterEntities/constants';
+import styles from './inputConditional.scss';
+
+const cx = classNames.bind(styles);
+
+export const InputConditional = ({
```

Comment:
Let's not use the same name as for previous component in old design. I suggest to rename it to `FieldTextConditional`.